rosidl_generator_cpp/resource/msg__struct.hpp.em

```diff
@@ -286,17 +280,17 @@ using @(spec.base_type.type) =
 
 // constants requiring out of line definition
 @[for c in spec.constants]@
-@[  if c.type not in ['byte', 'int8', 'int16', 'int32', 'int64', 'char', 'uint8', 'uint16', 'uint32', 'uint64']]@
 template<typename ContainerAllocator>
 const @(MSG_TYPE_TO_CPP[c.type])
 @(spec.base_type.type)_<ContainerAllocator>::@(c.name) =
-@[    if c.type == 'string']@
+@[  if c.type == 'string']@
   "@(escape_string(c.value))";
-@[    elif c.type == 'bool']@
+@[  elif (c.type in ['bool', 'byte', 'int8', 'int16', 'int32', 'int64', 'char'])]@
```

Comment:
Nitpicks: Python conditions shouldn't be wrapped in parenthesis.

Using a tuple rather than a list is preferred when the content isn't modified. Same below.